packages/apollo-react-relay-duct-tape/src/types.ts

```diff
@@ -2,8 +2,13 @@ export interface Variables {
   [name: string]: any;
 }
 
+export interface Context {
+  [name: string]: any;
+}
+
 export interface OperationType {
   readonly variables: Variables;
+  readonly context?: Context;
```

Comment:
Can you please document in every place where this option can be surfaced to say that this should be avoided when possible, as it will not be compatible with Relay's APIs.

Reply:
done.